feature-settings-impl/src/main/java/io/novafoundation/nova/feature_settings_impl/domain/CloudBackupSettingsInteractor.kt

```diff
@@ -38,7 +43,7 @@ class RealCloudBackupSettingsInteractor(
                 cloudBackupFacade.applyNonDestructiveCloudVersionOrThrow(cloudBackup, BackupDiffStrategy.syncWithCloud())
 
                 Unit
-            }.finally {
+            }.onSuccess {
```

Comment:
I think it should be `finally` - we probably want to display updated timestamp even on failed sync